workflow/scripts/collect-stratifications-fp-fn.py

```
@@ -0,0 +1,56 @@
+import sys
+
+sys.stderr = open(snakemake.log[0], "w")
```

Ruff (0.8.0)

Use a context manager for opening files

(SIM115)